defs/scenarios/storage/ceph/ceph-mon/eol.yaml

```diff
@@ -0,0 +1,12 @@
+checks:
+  is_eol:
+    requires:
+      property:
+        path: hotsos.core.plugins.storage.ceph.CephChecksBase.days_to_eol
+        ops: [[le, 0]]
+conclusions:
+  is-eol:
+    decision: is_eol
+    raises:
+      type: CephWarning
+      message: This version of Ceph is EOL
```

Comment:
try this:

message: >-
This node is running a version of Ceph that is End of Life (release={release}) which means it has limited support and is likely not receiving updates anymore. Please consider upgrading to a newer release.
format-dict:
release: hotsos.core.plugins.storage.ceph.CephChecksBase.release_name